### DISCUSSION TITLE
Atom feed tokens changed

### DISCUSSION THREAD

**Discussion by jean** [community/community#24441] [Apps, API and Webhooks]
<p>Hi all</p>
<p>Mystery solved: someone in the team updated their password, so the tokens changed.</p>
<p>===== Original message =====</p>
<p>I consume the atom feeds for commits and for releases from a couple of services such as Zapier.</p>
<p>This worked great for a couple of years, but a few days ago all the auth tokens on the feeds changed.</p>
<p>The old URLs looked like this: <a href="https://github.com/org/repo/commits/master.atom?token=AAFLQGm...F_wA" rel="nofollow noopener">https://github.com/org/repo/commits/master.atom?</a></p>
<p>The new URLs are looking like this: <a href="https://github.com/org/repo/commits/master.atom?token=AAFLQBD...TjwA==" rel="nofollow noopener">https://github.com/org/repo/commits/master.atom?</a></p>
<p>[token edited by moderator]</p>
<p>So (1) the token changed, and (2) it’s now necessary to include the final <code>==</code> (and not URL-encoded).</p>
<p>Did this bite anyone else?</p>
<p>Is this documented anywhere?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/jean">@jean</a>,</p>
<p>Thanks for coming back with the update! I’ve edited the token portion of your URL’s above as this information is sensitive, I’d suggest you delete that token and generate a new one to be 100% safe.</p>
<p>Thanks!</p>
<p>Andrea</p>

**Comment by jean** [comment, marked as answer]
<p>Wow even with the middle 30+ characters elided? Thanks!</p>
<p>In order to have an accepted answer, again: “Mystery solved: someone in the team updated their password, so the tokens changed.”</p>

**Comment by hughpv** [comment]
I started getting 404 on the commits feed for my private repository. Sure enough, I recently changed my password. But after a whole lot of looking, I can't figure out where to generate a new token. Can anyone help?
